bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "landing-upplication",
   "dependencies": {
     "jquery": "~2.1.4",
-    "materialize": "~0.97.8",
+    "materialize": "^0.100.2",
```

Comment:
modals dont trigger maybe there are some changes in the materialize framework between the 0.97 and the 0.100?
Check: https://materializecss.com/modals.html#!